KaijuKiller/Assets/Scripts/PlayerMove.cs

```diff
@@ -6,20 +6,26 @@
 public class PlayerMove : MonoBehaviour
 {
     [SerializeField]
-    private float moveSpeed = 10;
+    private float moveSpeed;
 
     [SerializeField]
     private Rigidbody rb;
 
+    [SerializeField]
+    private float jumpSpeed;
+
     private int bounds;
     private bool isLaneChanging;
+    private bool isJumping;
```

Comment:
Consider an enumeration for falling/standing/etc. Adds a decent level of modularity to your code once you expand past two states.

Reply:
that might just fix the bug